src/util/getAttributeValue.ts

```diff
@@ -7,6 +8,6 @@ export const getAttributeValue = (element: ElementAst, property: string) => {
     return attr.value;
   }
   if (input) {
-    return (<any>input.value).ast.value;
+    return (<any>input.value).ast.value || PROPERTY;
```

Comment:
Let's try to avoid these `any` casts.

You can do something like this instead:

```
if (!input || !(input.value instanceof ASTWithSource) || !(input.value.ast instanceof LiteralPrimitive)) {
  return null/undefined;
}

return input.value.ast.value;
```